### PR TITLE
User endpoints

### PR DESCRIPTION
**Pull request by PatrykJanas27**
add 3 enpoints for renting item and returning item by user

### CODE REVIEW

File: src/main/java/pl/creative/rental_server/core/features/moderator/item/ItemService.java [hunk 1]
```diff
@@ -224,6 +224,18 @@ public List<GetItemHistoryDto> getAllHistoryOfItem(String itemId) {
         }
     }
 
+    public List<GetItemDto> getAllItemsInDefaultPlace() {
```
Comment (adwas3213) [inline src/main/java/pl/creative/rental_server/core/features/moderator/item/ItemService.java]: Maybie getAllItemsFrom...

File: src/main/java/pl/creative/rental_server/core/features/user/RentUserService.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+@Slf4j
+public class RentUserService {
+    private final AccountRepository accountRepository;
+    private final RentItemService rentItemService;
+    private final ItemService itemService;
+
+    public void rentItemForMe(String itemId, String commentToEvent) {
+        log.info("Renting an item by user");
+        String accountEmail = SecurityContextHolder.getContext().getAuthentication().getName();
+        log.info("User account email: {}", accountEmail);
```
Comment (adwas3213) [inline src/main/java/pl/creative/rental_server/core/features/user/RentUserService.java]: Merge 2 Loggers into 1

**Review by adwas3213 — APPROVED** [review]

File: src/main/java/pl/creative/rental_server/core/features/moderator/item/ItemService.java (hunk 1 above)
Comment (PatrykJanas27) [reply src/main/java/pl/creative/rental_server/core/features/moderator/item/ItemService.java]: yeah, done

File: src/main/java/pl/creative/rental_server/core/features/user/RentUserService.java (hunk 2 above)
Comment (PatrykJanas27) [reply src/main/java/pl/creative/rental_server/core/features/user/RentUserService.java]: if you say so, done

**Comment by PatrykJanas27** [comment]
feat: add 3 enpoints for renting item and returning item by user

**Pull request merged**